### PR TITLE
feat: Added internal ingress 

### PR DESCRIPTION
**Pull request by amanpruthi**

### CODE REVIEW

File: charts/operator-wandb/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: operator-wandb
 description: A Helm chart for deploying W&B to Kubernetes
 type: application
-version: 0.11.5
+version: 0.13.5
```
Comment (wandb-zacharyblasczyk) [inline charts/operator-wandb/Chart.yaml:5]: 0.12.0?

File: charts/operator-wandb/values.yaml [hunk 2]
```diff
@@ -90,6 +90,22 @@ global:
...
+  tls: []
+  additionalHosts: []
+  class: ""
+  issuer:
+    create: false
+    provider: ""
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: support@wandb.com
+
+ingress_internal:
```
Comment (wandb-zacharyblasczyk) [inline charts/operator-wandb/values.yaml:107]: We don't use `_` I think camelCase is that is being standardized on. 

Also, what is the ing

Comment (wandb-zacharyblasczyk) [reply charts/operator-wandb/values.yaml:107]: Also, if this is for privatLink, maybe we should call this 
```
privatLinkIngress:
  create: false
...
```

or 
```
offlineIngress:
  create: false
...
```

We should probably document that this ingress is only needed for GCP/Azure/Bare metal?

**Review by wandb-zacharyblasczyk — COMMENTED** [review]
@gls4 @nfoucha @jsbroks @adityachoudhari26

File: charts/operator-wandb/templates/ingress.yaml [hunk 3]
```diff
@@ -80,3 +81,64 @@ spec:
               number: 8082
   {{- end }}
 {{- end }}
+{{- end }}
+---
+{{- if .Values.gcpPrivatLinkIngress.create }}
```
Comment (jsbroks) [inline charts/operator-wandb/templates/ingress.yaml:86]:
```suggestion
{{- if .Values.ingress.secondary.create }}
```

Lets generalize and call it `secondary`

File: charts/operator-wandb/templates/ingress.yaml [hunk 4]
```diff
@@ -80,3 +81,64 @@ spec:
...
+            port: 
+              number: 8080
+            {{- end }}
+      - pathType: Prefix
+        path: /console
+        backend:
+          service:
+            name: {{ $.Release.Name }}-console
+            port:
+              number: 8082
```
Comment (jsbroks) [inline charts/operator-wandb/templates/ingress.yaml:142]: Could we move this to a tpl and import that here? This way we only need to modify it in one place

**Review by jsbroks — CHANGES_REQUESTED** [review]

File: charts/operator-wandb/values.yaml [hunk 5]
```diff
@@ -116,6 +117,21 @@ ingress:
     provider: ""
     server: https://acme-v02.api.letsencrypt.org/directory
     email: support@wandb.com
+  secondary:
+    create: true
```
Comment (jsbroks) [inline charts/operator-wandb/values.yaml:121]:
```suggestion
    create: false
```

**Review by jsbroks — APPROVED** [review]

**Review by jsbroks — APPROVED** [review]

File: charts/operator-wandb/templates/ingress.yaml [hunk 6]
```diff
@@ -54,29 +55,54 @@ spec:
...
+  labels:
+    {{- include "wandb.commonLabels" . | nindent 4 }}
+    {{- if .Values.ingress.secondary.labels -}}
+    {{-   toYaml .Values.ingress.secondary.labels | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- if .Values.ingress.secondary.issuer.create }}
+    "cert-manager.io/issuer": "{{ .Release.Name }}-issuer"
+    "cert-manager.io/acme-challenge-type": "http01"
+    "kubernetes.io/ingress.allow-http" : "false"
```
Comment (zacharyblasczyk) [inline charts/operator-wandb/templates/ingress.yaml:82]: Typo?

File: charts/operator-wandb/values.yaml [hunk 7]
```diff
@@ -116,6 +117,21 @@ ingress:
     provider: ""
     server: https://acme-v02.api.letsencrypt.org/directory
     email: support@wandb.com
+  secondary:
+    create: false
+    install: true
```
Comment (zacharyblasczyk) [inline charts/operator-wandb/values.yaml:122]: Unused?

File: charts/operator-wandb/values.yaml [hunk 8]
```diff
@@ -116,6 +117,21 @@ ingress:
...
+    annotations: {}
+    labels: {}
+    tls: []
+    additionalHosts: []
+    class: ""
+    issuer:
+      create: false
+      provider: ""
+      server: https://acme-v02.api.letsencrypt.org/directory
+      email: support@wandb.com
```
Comment (zacharyblasczyk) [inline charts/operator-wandb/values.yaml:134]: Do these need to be wired up?

**Review by zacharyblasczyk — COMMENTED** [review]
Looks like a bit of copy paste.

**Pull request merged**